src/tree/registerTrees.ts

```diff
@@ -77,6 +78,15 @@ export function registerTrees(): void {
         await ext.volumesTree.refresh(node);
     });
 
+    ext.contextsRoot = new ContextsTreeItem(undefined);
+    const contextsLoadMore = 'vscode-docker.contexts.loadMore';
+    ext.contextsTree = new AzExtTreeDataProvider(ext.contextsRoot, contextsLoadMore);
+    ext.contextsTreeView = vscode.window.createTreeView('vscode-docker.views.dockerContexts', { treeDataProvider: ext.contextsTree, canSelectMany: true });
+    ext.context.subscriptions.push(ext.contextsTreeView);
+    ext.contextsRoot.registerRefreshEvents(ext.contextsTreeView);
```

Comment:
As an alternative to automatic refresh, I'd suggest letting Docker Context Manager trigger the refresh here. Additionally, since the refresh rate is default 2 seconds (with a setting), it would be more appropriate to follow the context manager's refresh rate of 20 seconds.

Reply:
Actually, I'm not sure Docker Context Manager would notice if a context is added but not activated, so that's not the best approach. Nevertheless, 2 seconds is definitely way too fast a pace for this, so we should do refresh slower. For example, on my machine, it takes about 5 seconds to do `docker context ls`. Globally the average seems to be about 1 second.

Reply:
Good point. I don't think we need 2 second refresh. Now using 20 seconds.